### ISSUE TITLE
OpenAPI does not recognise default values in data models

### ISSUE THREAD

**Issue by artdgn** [tiangolo/fastapi#1082]
### Describe the bug

When using a data model for a post endpoint the default values are not shown correctly in `/docs` UI, but are shown correctly in `/redoc` UI.

### To Reproduce

```Python
import fastapi
import pydantic

app = fastapi.FastAPI()


class Item(pydantic.BaseModel):
    name: str = pydantic.Field(default='a')
    description: str = '3'


@app.put("/")
def func(item: Item):
    return {}
```

- In `/docs`, the default values displayed **incorrectly**:
```json
{
  "name": "string",
  "description": "string"
}
```

- in `/redoc`, the default values are displayed correctly:
```json
{
    "name": "a",
    "description": "3"
}
```

### Environment

- OS: Linux, Ubuntu
- FastAPI Version `0.52.0`

- Python version `3.6.9`

**Comment by Dustyposa** [comment]
`/docs` swagger display the default value with schema.

**Comment by JohnPaton** [comment]
There is a difference between `default` and `example`.

```python
class Item(pydantic.BaseModel):
    name: str = pydantic.Field(default='a', example='a')
    description: str =pydantic.Field(default='3', example='3')
```
will do what you want. 

Might be a nice feature to use the `default` as an `example` if `example` isn't specified

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

If that solves the original problem, then you can close this issue :heavy_check_mark: 

You might also want to check the docs on extra examples: https://fastapi.tiangolo.com/tutorial/schema-extra-example/

**Comment by artdgn** [comment]
Thanks for looking into it, both the explanation (on the difference between example and default) and the suggestion to take it up with OAS / pydantic is helpful for someone who will find this issue after encountering this problem.

However, it won't:
 - Prevent future confusion for users who will intuitively expect pythonic behaviour by specifying one value (the one specified in the typing annotation), instead of using the Field class and specifying both verbosly and repetitively for every field. 
 - It doesn't solve the semantic inconsistency between `/docs` and `/redoc` UIs demonstrated in the OP.

**Comment by tiangolo** [comment]
Yeah, that would actually be a feature request for Swagger UI, not FastAPI.
